### PR TITLE
Bug/1205 dc banner appers too many times

### LINKED ISSUE #1205: DataCloud sign in screen appears one to many times
DataCloud sign in/welcoming screen appears while opening jobs as well.

### PR DESCRIPTION
**Pull request by ClaudiaPHI**
Fixes #1205

### CODE REVIEW

**Comment by kaspersorensen** [comment]
I would like to suggest an alternative solution: The dialog should not be tied to the ```AnalysisJobBuilderWindow``` _at all_. It has nothing to do with showing that window - it was a wrong idea to begin with. Rather it only has something to do with application startup. So the instruction to do it (without needing any counter or anything like that) should be done in the ```Bootstrap``` class which eventually also opens the (first) ```AnalysisJobBuilderWindow```

**Comment by kaspersorensen** [comment]
A good side-effect of that is also that we decrease the amount of code in ```AnalysisJobBuilderWindow``` - which is already a too complex class as it is.

**Comment by ClaudiaPHI** [comment]
oki doki

**Comment by ClaudiaPHI** [comment]
Moved DataCloud banner to ```Bootstrap``` class.

File: desktop/ui/src/main/java/org/datacleaner/bootstrap/Bootstrap.java [hunk 1]
```diff
@@ -249,8 +251,17 @@ public void run() {
                 });
             }
 
-            final WindowContext windowContext = injector.getInstance(WindowContext.class);
-
+            WidgetUtils.invokeSwingAction(new Runnable() {
+                @Override
+                public void run() {
+                    if (DataCloudLogInWindow.isRelevantToShow(userPreferences, configuration)) {
+                        DataCloudLogInWindow dataCloudLogInWindow = new DataCloudLogInWindow(configuration,
```
Comment (LosD) [inline desktop/ui/src/main/java/org/datacleaner/bootstrap/Bootstrap.java]: final

**Comment by LosD** [comment]
Nice and simple. LGTM (except a single missing final)

**Pull request merged**